server/src/main/java/org/elasticsearch/action/admin/cluster/stats/ClusterStatsResponse.java

```diff
@@ -137,6 +145,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
 
         if (CCS_TELEMETRY_FEATURE_FLAG.isEnabled()) {
             builder.startObject("ccs");
+            if (remoteClustersStats != null && remoteClustersStats.isEmpty() == false) {
```

Comment:
I'd suggest we include the field if remote stats are requested but no remotes are available:

```suggestion
            if (remoteClustersStats != null) {
```

Reply:
OK it's more of a product question but I have no objection for it. @naj-h do you have any preference here?

Reply:
Discussed and resolved that `cluster` entry will be present if the remotes option is active, with empty object if there are no remotes.